cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/MaintenanceEventTest.java

```diff
@@ -101,7 +101,7 @@ public void testAddBrokerEvent()
                                                           ANOMALY_DETECTION_TIME_MS_OBJECT_CONFIG, MOCK_TIME_MS);
 
     // Expect mocks.
-    EasyMock.expect(_mockKafkaCruiseControl.config()).andReturn(_config).times(3);
+    EasyMock.expect(_mockKafkaCruiseControl.config()).andReturn(_config).atLeastOnce();
```

Comment:
(applies to similar uses in this PR on this and other tests) Can we avoid dropping `times()` in tests?